frontend/helpers/issues.ts

```diff
@@ -15,34 +15,50 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { get, omit } from 'lodash';
+import { get } from 'lodash';
 import { getAPIUrl } from '../services/api';
 import { STATUSES_COLOURS, STATUSES_ICONS, STATUSES } from '../constants/issues';
 import { isAdmin, hasPermissions, PERMISSIONS } from './permissions';
-import { sortByDate } from './sorting';
+
+const renameFieldIfExists = (issue, fieldName, newFieldName) => {
+	if (issue[fieldName]) {
+		issue[newFieldName] = issue[fieldName];
+	}
+};
 
 export const prepareIssue = (issue, jobs = []) => {
-	const thumbnail = getAPIUrl(issue.thumbnail);
-	const descriptionThumbnail = issue.viewpoint && issue.viewpoint.screenshot
+	if (issue.thumbnail) {
+		const thumbnail = getAPIUrl(issue.thumbnail);
+		issue = {...issue, thumbnail};
+	}
+
+	const descriptionThumbnail = get(issue, 'viewpoint.screenshot')
```

Comment:
Is `get` necessary here?
`get` is much slower than the native solution: https://jsperf.com/get-vs-native